sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_funnel_v1/query.sql

```diff
@@ -48,19 +49,20 @@ store_stats AS (
 ),
 _new_profiles AS (
   SELECT
-    submission_date AS `date`,
-    country,
-    SUM(new_profiles) AS new_profiles,
+    first_seen_date AS `date`,
+    first_reported_country AS country,
+    COUNT(*) AS new_profiles,
   FROM
-    firefox_ios.active_users_aggregates
+    firefox_ios.firefox_ios_clients
   WHERE
-    submission_date >= '2022-01-01'
+    first_seen_date = DATE_SUB(@submission_date, INTERVAL 7 DAY)
     AND channel = "release"
   GROUP BY
     `date`,
     country
 )
 SELECT
+  @submission_date AS submission_date,
```

Comment:
I suggest leaving out this date. It doesn't have any business meaning and might cause confusion.

Reply:
I can, but then I am not able to partition this table.